sdk/samples/iot/hub/src/paho_iot_hub_twin_example.c

```diff
@@ -141,37 +171,9 @@ int main()
     return rc;
   }
 
-  printf("\nSubscribed to topics.\n");
-  printf("\nWaiting for activity:\n"
-         "Press 'g' for device to request twin document from service.\n"
-         "Press 'r' for device to send \"device_count\" reported property to service.\n"
-         "\t\t\t\"device_count\" will then locally increment.\n"
-         "[Press 'q' to quit]\n\n");
-
-  int input;
-  while (1)
-  {
-    input = getchar();
-    if (input != '\n')
-    {
-      switch (input)
-      {
-        case 'g':
-          send_get_twin();
-          break;
-        case 'r':
-          report_property();
-          increment_property_count();
-          break;
-        default:
-          break;
-      }
-      if (input == 'q')
-      {
-        break;
-      }
-    }
-  }
+  get_twin_document();
+  send_reported_property();
+  get_desired_property();
```

Comment:
```suggestion
  receive_desired_property();
```
"get" sounds as though it is actively asking for it and expecting something in return. "receive" makes it more explicit it is waiting for something.

Reply:
totally.